frontend/decorate.js

```diff
@@ -130,14 +130,14 @@ module.exports = function (options: Options, Component: any): any {
 function arrayDiff(array, oldArray) {
   var names = new Set();
   var missing = [];
-  for (var i = 0; i < array.length; i++) {
-    names.add(array[i]);
+  for (var ai = 0; ai < array.length; ai++) {
```

Comment:
If flow supported `let` already, we could use that here...

Otherwise, I'd prefer `i`, `j`, `k` as more traditional.